resources/flaky-github-comment-markdown.template

```diff
@@ -2,7 +2,7 @@
 ## :bug: Flaky test report
 :snowflake: The following tests failed but also have a history of flakiness and may not be related to this change: 
     <% flakyTests?.each{ k,v -> %>
-* **Name**: `${k}` ${ (v?.trim()) ? "reported in the issue #" + v : 'has not been reported yet.'}<%}%>
+* **Name**: `${k}` ${ (v?.trim()) ? "reported in the issue " + v : 'has not been reported yet.'}<%}%>
```

Comment:
`gh` returns the URL therefore # is not required anymore to link with issues correctly